src/lib/buildVoteGraph.ts

```diff
@@ -1,17 +1,18 @@
 import { IPoll, IVoter } from '../definition';
 
-const filled = '█';
-const empty = ' ';
-const width = 20;
+const empty = '  ';
+const width = 10;
 
 const format = (num) => new Intl.NumberFormat('en-US', {
     minimumFractionDigits: 2,
     maximumFractionDigits: 2,
 }).format(num);
 
-export function buildVoteGraph(votes: IVoter, totalVotes: IPoll['totalVotes']) {
+export function buildVoteGraph(votes: IVoter, totalVotes: IPoll['totalVotes'], maxVoteIndex: boolean) {
     const percent = totalVotes === 0 ? 0 : votes.quantity / totalVotes;
 
+    const filled = maxVoteIndex ? '🟦': '⬛';
```

Comment:
Why are we changing the emoji? can include the existing emoji █ ?

Reply:
The existing one was monochrome (results in indistinguishable winner with black bars in bar graph). Adding these new ones clearly distinguishes winning options from the non-winning ones. Color chosen to match the Twitter poll.